src/Sulu/Bundle/ContentBundle/Resources/public/scss/modules/general.scss

```diff
@@ -2,14 +2,10 @@
     width: 95px;
 }
 
-#content-form {
+#content-form,
+.content-form {
```

Comment:
Are there already so many bundles using this? Then I would at least remove the `content-` part from the class, because it doesn't make sense in another context. The only other css definition based on that I could find was in the `teaser-selection`. Maybe we can rename it there? Or we call it something like `sulu-form`.